data/json/professions.json

```diff
@@ -4149,5 +4149,33 @@
       }
     },
     "flags": [ "SCEN_ONLY" ]
+  },
+  {
+    "type": "profession",
+    "id": "road_worker",
+    "name": "Road Worker",
+    "description": "You have a road roller.  Nothing can stop you.  You are god.",
+    "points": 10,
+    "vehicle": "road_roller",
+    "skills": [ { "level": 3, "name": "driving" } ],
+    "items": {
+      "both": {
+        "items": [
+          "hat_hard",
+          "glasses_safety",
+          "vest_safety",
+          "gloves_work",
+          "socks",
+          "technician_pants_gray",
+          "technician_shirt_gray",
+          "tool_belt",
+          "boots"
+        ],
+        "entries": [ { "item": "shovel", "custom-flags": "auto_wield" } ]
+      },
+      "male": [ "boxer_shorts" ],
+      "female": [ "bra", "panties" ]
+    },
+    "traits": "PSYCHOPATH"
```

Comment:
Is this trait really needed for this profession?

Reply:
Not really needed as the name goes, but it fits the description and the rest quite well